# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/awesome/hotfix-no-check.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- awesome-4.3/CMakeLists.txt.vanilla	2025-11-16 22:24:17.227230582 +0200
+++ awesome-4.3/CMakeLists.txt	2025-11-16 22:25:03.742324971 +0200
@@ -418,6 +418,7 @@
     DEPENDS generated_icons generate_awesomerc ${PROJECT_AWE_NAME})
 #add_dependencies(check check-themes)
 
+if(0)
 add_custom_target(check-requires
     lua "${CMAKE_SOURCE_DIR}/build-utils/check_for_invalid_requires.lua"
     WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
@@ -457,6 +458,8 @@
 add_dependencies(check check-qa check-examples)
 # }}}
 
+endif()
+
 INCLUDE(${CMAKE_SOURCE_DIR}/Packaging.cmake)
 
 # vim: filetype=cmake:expandtab:shiftwidth=4:tabstop=8:softtabstop=4:textwidth=80:foldmethod=marker
